Makefile

```diff
@@ -9,16 +9,19 @@ generate:
 	operator-sdk generate k8s
 
 build:
-	go build -ldflags "-X github.com/openshift/cluster-image-registry-operator/version.Version=$$(git describe --tags --always --dirty)" "./cmd/$(PROG)"
+	./tmp/build/build.sh
 
 build-image:
 	docker build -t "$(IMAGE)" .
 
-build-devel-image: build
-	docker build -t "$(IMAGE)" -f Dockerfile-dev .
+build-devel-image:
+	operator-sdk build "$(IMAGE)"
 
 test:
 	go test ./...
 
+test-e2e:
+	operator-sdk test --test-location=./test/e2e
```

Comment:
what does this actually require to run?  does it need a docker socket?

Reply:
It requires kubeconfig with admin capabilities (it is going to deploy the operator and create namespaces).

Also it requires [deploy/sa.yaml, deploy/rbac.yaml and deploy/operator.yaml](https://github.com/operator-framework/operator-sdk/blob/d23344453ec6124f1555fafc2e66230f98410bbb/commands/operator-sdk/cmd/test.go#L58) or a special file with resources that should be created in test namespaces.

docker socket isn't required.

Reply:
ok